email_notification/helpers/email_builder_base.py

```diff
@@ -48,7 +46,7 @@ def _get_common_kwargs(cls, language) -> dict[str, Any]:
             "language": language,
             "varaamo_ext_link": cls._get_varaamo_ext_link(language),
             "feedback_ext_link": cls._get_feedback_ext_link(language),
-            "current_year": cls._get_current_year(),
+            "current_year": local_datetime().year,
         }
 
     @staticmethod
```

Comment:
Above: `_get_common_kwargs` is missing typing for the `language` parameter.

Below: `_get_varaamo_ext_link` and `_get_feedback_ext_link` `language` parameters are not nullable.

Reply:
Fixed type hints